_getj2me

```diff
@@ -0,0 +1,93 @@
+#! /usr/bin/env node --harmony
+
+const APP_HOST = 'marketplace.firefox.com';
+const APP_PATH = '/app/f2ed31a0-05ed-416b-ac0d-0f26db063839/manifest.webapp';
+
+const PACKAGE_PATH = 'apps/j2me/';
+const PREFIX = '[j2me]';
+
+var https = require('https');
+var exec = require('child_process').exec;
+var spawn = require('child_process').spawn;
+var url = require('url');
+var fs = require('fs');
+
+var package;
+
+var begin = function() {
+  console.log(PREFIX, 'Starting app download process.');
```

Comment:
nit: I would opt to just use the node debug module which should handle the prefixes for you?